CONTRIBUTING.md

```diff
@@ -207,9 +211,74 @@ To build the documentation locally:
 
 1. Navigate to the `docs` directory.
 2. Run `make html`.
-3. Open "_build/html/index.html" in your browser.
+3. Open "\_build/html/index.html" in your browser to inspect the documentation.
```

Comment:
Is this backslash at the start of the URL intentional?

Reply:
Hmm it was added in by my autoformatter automatically, to escape the underscore. I think it renders fine on GH

![image](https://github.com/shap/shap/assets/30731072/c903b91f-f348-4132-b5e2-cacd3b795042)